pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -151,3 +170,24 @@ func (ci *CloudInfo) getNetwork(networkName string) (*networks.Network, error) {
 
 	return network, nil
 }
+
+func (ci *CloudInfo) getImage(imageName string) (*images.Image, error) {
+	if imageName == "" {
+		return &images.Image{}, nil
```

Comment:
Question: Why do you return a pointer to an empty image? I expected to see `return nil nil` here

Reply:
This is an absolutely useless check and should be removed. when we call `getImage` we check that the image name is non-empty.
Thank you for pointing at this!